src/Features/CSharpTest/ExtractMethod/ExtractMethodTests2.cs

```diff
@@ -5453,4 +5481,389 @@ static void NewMethod()
+
+                public static async Task Goo(string value)
+                {
+                    [|var anotherValue = "GooBar";
+                    var customer = await _repository.GetValue(value.Length);|]
+
+                    Console.Write(customer.Id);
+                    Console.Write(anotherValue);
+                }
+            }
+            """,
+            """
+            using System;
+            using System.Threading.Tasks;
+            
+            class Customer
+            {
+                public int Id;
+            }
+            
+            class Repository
+            {
+                private static readonly Repository _repository = new();
+                public Task<Customer> GetValue(int i) => null!;
+            
+                public static async Task Goo(string value)
+                {
+                    (string anotherValue, Customer customer) = await {|Rename:NewMethod|}(value);
+            
+                    Console.Write(customer.Id);
+                    Console.Write(anotherValue);
+                }
+
+                private static async Task<(string anotherValue, Customer customer)> NewMethod(string value)
+                {
+                    var anotherValue = "GooBar";
+                    var customer = await _repository.GetValue(value.Length);
+                    return (anotherValue, customer);
+                }
+            }
+            """,
+            options: ExplicitTypeEverywhere());
+
+    [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/64597")]
+    public Task TestMultipleOutTuple_ImplicitForBuiltInTypes()
+        => TestInRegularAndScriptAsync(
+            """
+            using System;
+            using System.Threading.Tasks;
+
+            class Customer
+            {
+                public int Id;
+            }
+
+            class Repository
+            {
+                private static readonly Repository _repository = new();
+                public Task<Customer> GetValue(int i) => null!;
+
+                public static async Task Goo(string value)
+                {
+                    [|var anotherValue = "GooBar";
+                    var customer = await _repository.GetValue(value.Length);|]
+
+                    Console.Write(customer.Id);
+                    Console.Write(anotherValue);
+                }
+            }
+            """,
+            """
+            using System;
+            using System.Threading.Tasks;
+            
+            class Customer
+            {
+                public int Id;
+            }
+            
+            class Repository
+            {
+                private static readonly Repository _repository = new();
+                public Task<Customer> GetValue(int i) => null!;
+            
+                public static async Task Goo(string value)
+                {
+                    (var anotherValue, Customer customer) = await {|Rename:NewMethod|}(value);
```

Comment:
I'd definitely prefer a `var (..., ...)` deconstruction here, since the user declared both `anotherValue` and `customer` with var.

```C#
var (anotherValue, customer) = await {|Rename:NewMethod|}(value);
```